cbioportal_etl/scripts/diff_studies.py

```diff
@@ -1,22 +1,25 @@
 #!/usr/bin/env python3
-"""
-Script to check a CURRENT study on a current cbioportal instance for differences against a local UPDATE build.
+"""Script to check a CURRENT study on a current cbioportal instance for differences against a local UPDATE build.
+
 Outputs files that are the foundation for incremental updates and clinical data QC
 """
 
 import argparse
-from urllib.parse import urlparse
+import csv
 import glob
-import sys
 import os
-import csv
+import sys
 import typing
-import re
+from urllib.parse import urlparse
+
 import requests
 
 
-def parse_file(file_path: str, header_symbol: str | None = None, column_names: bool = False) -> tuple[list[str] | None, list[str] | None, list[str]]:
+def parse_file(
+    file_path: str, header_symbol: str | None = None, column_names: bool = False
```

Comment:
why does this end up as one line rather than each input getting its own line?

Reply:
🤷‍♂️